Formula/t/tmux-sessionizer.rb

```diff
@@ -4,6 +4,7 @@ class TmuxSessionizer < Formula
   url "https://github.com/jrmoulton/tmux-sessionizer/archive/refs/tags/v0.4.4.tar.gz"
   sha256 "9dfbe99a3c1fe7f48be0c1ab9056e49f36c4f85d023e24f874254f6791a9894e"
   license "MIT"
+  revision 1
```

Comment:
```suggestion
```

Let's drop the revision bump to avoid mandating users to install a new version. We only do this when necessary, and those who'd like to get the completions can do `brew reinstall tmux-sessionizer`.

If you apply the suggestion on GitHub UI, please squash the commits afterwards (following our "one commit per formula" rule). Thanks!